packages/frontend-2/components/project/page/team/Dialog.vue

```diff
@@ -21,6 +25,8 @@ import type { ProjectPageTeamDialogFragment } from '~~/lib/common/generated/gql/
 import { graphql } from '~~/lib/common/generated/gql'
 import { useTeamDialogInternals } from '~~/lib/projects/composables/team'
 
+type OpenSectionType = 'invite' | 'access' | null
```

Comment:
Duplicate type (I saw the same one in Settings.vue), export this out from some kind of helper. Also, it might be easier to work with if you define it as an enum instead.